pkgs/tools/filesystems/mount-zip/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, fuse, boost, gcc, icu, libzip, pandoc
+, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "mount-zip";
+  version = "1.0.8";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "mount-zip";
+    rev = "v${version}";
+    sha256 = "+slN5eedSddYKgiNLq4KZ5BXwvGQw9859sVfkcIBYeo=";
+  };
+
+  nativeBuildInputs = [ boost gcc icu pandoc pkg-config ];
+  buildInputs = [ fuse libzip ];
+
+  makeFlags = [ "prefix=$(out)" ];
+
+  meta = with lib; {
+    inherit (src) homepage;
```

Comment:
```suggestion
    inherit (src.meta) homepage;
```

Reply:
Added the github page as a string instead - I was probably trying too hard with the metadata stuff. 😉